emails/2022-07-22.md

```diff
@@ -0,0 +1,46 @@
+This is Tim Sehn (tim@dolthub.com), Founder and CEO of DoltHub. We decided this week to ditch the automated or semi-automated emails you all were getting as users. We're a tiny company. We want to build a personal connection with as many of you as possible. Automated emails weren't helping with that. I'm going to write this weekly email to you all every Friday until I can't, which I hope is many years from now.
+
+# That belongs in a museum...
+
+We're making great progess on the [museum collections bounty](https://www.dolthub.com/repositories/dolthub/museum-collections). We're up to 10.6M objetcs in `main`.
```

Comment:
```suggestion
We're making great progess on the [museum collections bounty](https://www.dolthub.com/repositories/dolthub/museum-collections). We're up to 10.6M objects in `main`.
```

Reply:
"objects" is a little bland, "pieces of art, sculptures, and historical artifacts" or something?

Reply:
Agreed